src/_data/sidenav.yml

```diff
@@ -28,16 +28,12 @@
 
 - title: Samples & codelabs
   children:
+    - title: Sample apps on GitHub
+      permalink: https://github.com/flutter/samples/blob/master/INDEX.md
     - title: Cookbook
       permalink: /docs/cookbook
-    - title: UI samples catalog
-      permalink: /docs/catalog/samples
```

Comment:
So now we won't have any sidenav links into the "Sample App Catalog". Is this really what we want?

It seem like there are lots of pages under `/docs/catalog/samples`, but I'm unsure that they are all being references (from anywhere else in the site):

> <img width="241" alt="screen shot 2018-10-29 at 14 34 22" src="https://user-images.githubusercontent.com/4140793/47672317-10e0c000-db88-11e8-9824-882aab2776a9.png">

Reply:
It's what Andrew wants.

Reply:
@RedBrogdon, do you plan to incorporate those old examples into your GitHub examples? Can we delete those examples?

Reply:
It's possible the folks on the engineering team might feel differently since they're the ones that originally wrote those samples, but I believe we'd be better served by either creating similar content in the cookbook or creating a "basic Flutter UI components" sample app in flutter/samples that covered the same Widgets.

It's not that the content they contain is incorrect or anything -- more that they're off DevRel's radar in their current form and location, and wouldn't be getting the same maintenance from us or exposure in our get-the-word-out efforts.